packages/manager/src/features/CloudPulse/shared/CloudPulseAppliedFilter.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Chip, Typography } from '@linode/ui';
+import React from 'react';
+
+export interface CloudPulseAppliedFilterProps {
+  filters: {
+    [label: string]: string[];
+  };
+}
+export const CloudPulseAppliedFilter = (
+  props: CloudPulseAppliedFilterProps
+) => {
+  const { filters } = props;
+
+  return (
+    <Box
+      data-testid="applied-filter"
+      display="flex"
+      flexDirection={{ sm: 'row', xs: 'column' }}
+      flexWrap={{ sm: 'wrap' }}
+      maxHeight="70px"
+      mb={2}
+      mx={3}
+      overflow="auto"
+      pb={2}
+      rowGap={1.5}
+    >
+      {Object.entries(filters).map((data, index) => {
+        const label = data[0];
+        const filterValue = data[1];
+        return (
+          <React.Fragment key={index}>
+            <Typography
+              fontSize="14px"
+              key={label}
+              mr={1}
+              variant="h3"
+            >{`${label}:`}</Typography>
+            {filterValue.map((value, index) => {
+              return (
+                <Chip
+                  sx={{
+                    backgroundColor: '#f0f7ff',
+                    color: '#3d3d42',
+                    fontSize: '14px',
+                    mr: index === filterValue.length - 1 ? 4 : 1,
+                    px: 1,
+                    py: 0.5,
+                    width: { sm: 'fit-content', xs: '100%' },
+                  }}
```

Comment:
```suggestion
                  sx={{
                    backgroundColor: theme.tokens.color.Ultramarine[10],
                    color: theme.tokens.color.Neutrals.Black,
                    fontSize: '14px',
                    mr: index === filterValue.length - 1 ? 4 : 1,
                    px: 1,
                    py: 0.5,
                    width: { sm: 'fit-content', xs: '98%' },
                  }}
```

Yes, we should avoid using hardcoded color values moving forward. The values suggested here are the closest match I think we can use. 

I think we may want to keep `98%` instead of `100%`, just to leave a small gap between the right side of the chip and the vertical scrollbar.


> I've updated the color & width. Color I've used is theme.color.offblack & hardcoded it so that it'll not change when theme switches.

Using `theme.color.*` will definitely change when the theme switches, as those color values are theme-sensitive. If you want a color that remains consistent regardless of the theme mode, you should use `global` design color tokens via `theme.tokens.color.*`

Reply:
I was actually looking for similar color from themes that we can instead of hardcoded color. Thanks for suggesting it

Reply:
@pmakode-akamai  Any documentation available for these themes from where we can easily pick colors by visualizing? Can you please share if any available?

Reply:
there are 2 types of tokens as of now - `Global` (theme-agnostic) and `Alias` (theme-sensitive).

You can refer this repo https://github.com/linode/design-language-system, (or specifically [this file for light](https://github.com/linode/design-language-system/blob/main/dist/index.d.ts) and [this file for dark](https://github.com/linode/design-language-system/blob/main/dist/themes/dark/index.d.ts)) -- Generally, color values for `Global` tokens are the same in both dark and light files, but differ for `Alias` tokens.


Currently, we have categorized these tokens [here](https://github.com/linode/manager/blob/1072d546d7165e4a59aaaa753311a24a100cac50/packages/ui/src/foundations/themes/index.ts#L110) in the code comments (for dev reference)

Reply:
thanks